conda/models/match_spec.py

```diff
@@ -680,14 +680,14 @@ def __init__(self, value):
         super(FeatureMatch, self).__init__(self._convert(value))
 
     def _convert(self, value):
-        if isinstance(value, frozenset):
-            return value
-        elif not value:
+        if not value:
             return frozenset()
         elif isinstance(value, string_types):
-            return frozenset(value.replace(' ', ',').split(','))
+            return frozenset(f for f in (
```

Comment:
I mean, can we directly tests this specific method?

Reply:
I see. Yeah.  Doing that now.